stack_master.gemspec

```diff
@@ -20,8 +20,14 @@ Gem::Specification.new do |spec|
   spec.email         = ["steve@hodgkiss.me", "gstamp@gmail.com"]
   spec.summary       = %q{StackMaster is a sure-footed way of creating, updating and keeping track of Amazon (AWS) CloudFormation stacks.}
   spec.description   = %q{}
-  spec.homepage      = "https://github.com/envato/stack_master"
+  spec.homepage      = "https://opensource.envato.com/projects/stack_master.html"
   spec.license       = "MIT"
+  spec.metadata      = {
+    "bug_tracker_uri"   => "https://github.com/envato/stack_master/issues",
+    "changelog_uri"     => "https://github.com/envato/stack_master/blob/master/CHANGELOG.md",
+    "documentation_uri" => "https://www.rubydoc.info/gems/stack_master/#{spec.version}",
+    "source_code_uri"   => "https://github.com/envato/stack_master/tree/v#{spec.version}",
```

Comment:
Add some gemspec metadata. This'll become available from on the [Rubygems project page](https://rubygems.org/gems/stack_master) and via the `bundle` and `gem` command line tools after the next release.